docker/compose/indexer-postgres/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Aptos-Indexer, Postgres and pgAdmin panel
```

Comment:
this is a super nitty nit, but I'd prefer if you could name the file `README.md` for consistency.